_posts/2017-01-18-ruby-meets-rust.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Ruby meets Rust"
+date: 2017-01-18 19:00:00 MEST
+categories: meetup cologne
+links:
+    "Meetup.com": "https://www.meetup.com/RustCologne/events/235877954/"
+    "Cologne.rb": "http://www.colognerb.de/events/januar-meetup-2017"
+    "Planning discussion on Github": "https://github.com/Rustaceans/rust-cologne/issues/19"
+location: coworking-cologne
+talks:
+- title: "Using Rust to make Ruby faster"
```

Comment:
@badboy I haven't seen a final title or description so I went with this. Feel free to edit.